docs/sources/operations/storage/retention.md

```diff
@@ -235,24 +235,25 @@ Example configuration with GCS with a 28 day retention:
 schema_config:
   configs:
   - from: 2018-04-15
-    store: bigtable
+    store: tsdb
```

Comment:
leave the table manager examples as is given it's deprecated? I think it makes sense to leave these to use bigtable

Reply:
This belongs to the _Example Configuration_ section. The _Table Manager (deprecated)_ is a separate section.

Reply:
this is configuring retention using table manager. this won't work for tsdb index (or may work) :?
```
table_manager:
  retention_deletes_enabled: true
  retention_period: 672h # 28 days
```